xbmc/guilib/GUIIncludes.cpp

```diff
@@ -295,6 +300,127 @@ void CGUIIncludes::ResolveIncludesForNode(TiXmlElement *node, std::map<INFO::Inf
     node->FirstChild()->SetValue(ResolveConstant(node->FirstChild()->ValueStr()));
 }
 
+map<string, string> CGUIIncludes::GetParameters(const TiXmlElement *includeCall, const TiXmlElement *includeDef) const
+{
+  static const char paramNamespacePrefix[] = "p:";
+  map<string, string> params;
+  const TiXmlAttribute *attribute;
+  // add actual parameters from include call tag (these override defaults)
```

Comment:
Please add an empty line before this (same applies throughout the code) as it's pretty hard to read if there's no visual separation.